tensorboard/webapp/feature_flag/actions/feature_flag_actions.ts

```diff
@@ -43,3 +43,16 @@ export const featureFlagOverrideChanged = createAction(
     flags: Partial<FeatureFlags>;
   }>()
 );
+
+export const featureFlagOverridesReset = createAction(
+  '[FEATURE FLAG] Resetting feature flag overrides',
```

Comment:
Nit: Match tense of other actions "Resetting" => "Reset".

And the other one, too.